packages/@lwc/integration-karma/test/component/face-callbacks/index.spec.js

```diff
@@ -13,10 +16,38 @@ const createFormElement = () => {
     return container.shadowRoot.querySelector('form');
 };
 
-const createFaces = (tagName, ctor) => [
-    createElement(`face-${tagName}`, { is: ctor }),
-    createFaceUsingCec(`cec-face-${tagName}`, ctor.CustomElementConstructor),
-];
+const createFaceTests = (tagName, ctor, callback) => {
+    const scenarios = ['lwc.createElement', 'CustomElementConstructor'];
+
+    scenarios.forEach((scenario) => {
+        describe(scenario, () => {
+            const createFace = () => {
+                if (scenario === 'lwc.createElement') {
+                    return createFaceUsingLwcCreateElement(`face-${tagName}`, ctor);
+                } else {
+                    return createFaceUsingCec(`cec-face-${tagName}`, ctor.CustomElementConstructor);
+                }
+            };
+
+            callback(createFace, scenario);
+        });
+    });
+};
+
+const createFaceUsingLwcCreateElement = (tagName, ctor) => {
+    let elm;
+    const doCreate = () => {
+        elm = createElement(tagName, { is: ctor });
+    };
+    if (ENABLE_ELEMENT_INTERNALS_AND_FACE) {
+        doCreate();
```

Comment:
Could we use `process.env.SYNTHETIC_SHADOW_ENABLED` here instead of `process.env.NATIVE_SHADOW`?

Unless it's possible for `process.env.FORCE_NATIVE_SHADOW_MODE_FOR_TEST` and `process.env.SYNTHETIC_SHADOW_ENABLED` to be enabled at the same time.

Reply:
I kind of consider `process.env.NATIVE_SHADOW` to be the "default" choice for tests, and that `SYNTHETIC_SHADOW_ENABLED` is only needed for odd cases where you need to do something differently in `FORCE_NATIVE_SHADOW_MODE_FOR_TEST` mode.

Admittedly `FORCE_NATIVE_SHADOW_MODE_FOR_TEST` is kind of weird, so it's confusing. Maybe we should have better names here.